codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -115,7 +115,15 @@ public void operationComplete(ChannelFuture future) {
 
     private boolean isNotWebSocketPath(FullHttpRequest req) {
         String websocketPath = serverConfig.websocketPath();
-        return serverConfig.checkStartsWith() ? !req.uri().startsWith(websocketPath) : !req.uri().equals(websocketPath);
+        boolean checkNextUri = true;
+        if (req.uri().length() > websocketPath.length()) {
```

Comment:
`req.uri()` could be moved to a variable.

Reply:
Okay, thank you for reviewing.